### PR TITLE
Feature/custom comparison page

### PR DESCRIPTION
**Pull request by kfilip94**
This PR adds a blank "Custom comparison" page under the `/custom-compare` url, requires a `FEATURE_COMMITMENTS_OVERVIEW` flag to run. You can find this in the main menu.
[PIVOTAL](https://www.pivotaltracker.com/story/show/170676363)

image

### CODE REVIEW

File: app/javascript/app/routes/app-routes/app-routes.js [hunk 1]
```diff
@@ -192,6 +194,12 @@ export default [
     exact: true,
     headerGradient: HEADER_GRADIENTS.commitments
   },
+  FEATURE_COMMITMENTS_OVERVIEW && {
```
Comment (Bluesmile82) [inline app/javascript/app/routes/app-routes/app-routes.js]: The custom compare doesn't have a nav item on the menu. You access it through the compare all targets.

Comment (kfilip94) [reply app/javascript/app/routes/app-routes/app-routes.js]: I know, I added that for testing purpose, until the "compare all targets" page will be ready. Should I remove it anyway for not confusing our client?

Comment (Bluesmile82) [reply app/javascript/app/routes/app-routes/app-routes.js]: No, it's ok then. Thanks. We just have to remember

Comment (kfilip94) [reply app/javascript/app/routes/app-routes/app-routes.js]: ok, thank you :wink:

**Pull request merged**